Dart/src/com/jetbrains/lang/dart/ide/runner/server/vmService/DartVmServiceDebugProcess.java

```diff
@@ -147,7 +150,6 @@ public void logError(final String message) {
           return;
         }
 
-        getSession().getConsoleView().print("Error: " + message + "\n", ConsoleViewContentType.ERROR_OUTPUT);
```

Comment:
It depends on what kind of garbage can go here and how severe there warnings are.
Main point is that none of the predictable messages should go here. Only indicators of some unexpected problems.
On one hand the more visible is the message the more chances that user will report it to us. OTOH we do not want to bother user too much especially if functionality doesn't break because of it.
Truly speaking, LOG.warn() has little chances to reach developers. LOG.error has ore chances because errors have indication in EAPs.